### PR TITLE
fix: Update tsc to 2.x

### PR DESCRIPTION
**Pull request by seebees**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/random-source-browser/test/index.test.ts [hunk 1]
```diff
@@ -1,35 +1,36 @@
...
-  beforeEach(() => {
-    sinon.stub(webCryptoRandom, "randomValues");
-    sinon.stub(webCrypto, "supportsWebCrypto");
+    beforeEach(() => {
+      sinon.stub(webCryptoRandom, "randomValues");
   });
 
   afterEach(() => {
     (<sinon.SinonStub>webCryptoRandom.randomValues).restore();
-    // @ts-ignore
```
Comment (texastony) [inline packages/random-source-browser/test/index.test.ts:18]: Praise: Woot no more `ts-ignore`!

File: packages/sha1-browser/test/crossPlatformSha1.test.ts
Comment (texastony) [inline packages/sha1-browser/test/crossPlatformSha1.test.ts:1]: Question: Should we just delete these files?  
Or are we going to create a GHI to restore this?

**Review by texastony — APPROVED** [review]
Approve with Question

File: packages/sha1-browser/test/crossPlatformSha1.test.ts
Comment (seebees) [reply packages/sha1-browser/test/crossPlatformSha1.test.ts:1]: I don't like deleting tests.
I find it better to try and refactor to keep the tests than to just delete them.
This seems pretty low touch so this was my first cut

**Review by texastony — APPROVED** [review]

**Pull request merged**